AmplifyPlugins/DataStore/Tests/DataStoreHostApp/DataStoreStressTests/DataStoreStressTests.swift

```diff
@@ -48,13 +48,15 @@ final class DataStoreStressTests: DataStoreStressBaseTest {
             posts.append(post)
         }
         
+        let observeQuerySetup = asyncExpectation(description: "ObserveQuery setup done")
```

Comment:
why is this needed?

Reply:
datastore stress tests were failing due to timeout intermittently. i figured that observeQuery was not being setup at the right time since it starts a new task. after this change, the tests are running reliably.

Reply:
moved observe API out of the task